File: ABC/ABC375/A/main.cc
#include <iostream>
#include <string>

int main() {
  std::size_t n;
  std::string s;
  std::cin >> n >> s;

  int res = 0;
  for (std::size_t i = 0; i < n - 1; ++i) {
    if (s.substr(i, 3) == "#.#") ++res;
  }

  std::cout << res << "\n";
  return 0;
}

File: ABC/ABC375/A/main.go
package main

import "fmt"

func main() {
	var n int
	var s string
	fmt.Scan(&n, &s)

	res := 0
	for i := 0; i < n - 2; i++ {
		if s[i: i + 3] == "#.#" {
			res++
		}
	}

	fmt.Println(res)
}